packages/auth/src/authClient.ts

```diff
@@ -1,14 +1,19 @@
 import type { default as GoTrue, User as GoTrueUser } from 'gotrue-js'
 import type { Auth0Client as Auth0 } from '@auth0/auth0-spa-js'
 import type NetlifyIdentityNS from 'netlify-identity-widget'
+
+import { MagicSDKAdditionalConfiguration } from 'magic-sdk'
+import { SDKBase,  } from 'magic-sdk/dist/cjs/core/sdk'
```

Comment:
My guess is that you want to import the types here rather than the actual package.

Reply:
This may be the cause of your `error TS2307: Cannot find module 'react-native'`

Reply:
I did see a rouge comma. It's seems to just be a problem with pulling the correct types from the SDK. As it is already programmed in Typescript.

Reply:
Those types from Magic are crazy difficult to work with

Reply:
do you think its worth it to reach out to magic asking them the best way to pull the types?

Reply:
Totally!

Reply:
Hey there, I'm with the Magic SDK team. This would explain the `error TS2307: Cannot find module 'react-native'` to me. It's not ideal to import from the `dist/**` files directly because we strip out React Native dependencies. We'll need an easier way to get the constructor type. We have some pretty fancy typings to support our extensions architecture. I didn't consider this case when we added those in. I'll make sure we patch it from our side!

Reply:
A quick solution for now would be this:

```ts
import { Magic } from 'magic';

const foo: InstanceType<typeof Magic> = new Magic();
```

Reply:
magic-sdk@1.3.4. fixes this only got to solve the logout type issue.